packages/react-api/src/Api.tsx

```diff
@@ -162,12 +162,15 @@ async function loadOnReady (api: ApiPromise, injectedPromise: Promise<InjectedEx
 
   setDeriveCache(api.genesisHash.toHex(), deriveMapCache);
 
+  const isEthereum:boolean = (api.runtimeVersion.specName.eq('node-moonbeam') || api.runtimeVersion.specName.eq('moonbase-alphanet'));
```

Comment:
Currently, there is also the standalone version that is using `moonbeam-standalone`